### ISSUE TITLE
Map Generation Info Label should be moved to map corner

### ISSUE THREAD

**Issue by Starfire13** [dcs-liberation/dcs_liberation#1959]
### Affected versions

Development build

### Development build

6.0

### Description

The "Generated on x date/time by DCS Liberation" label appears at inconsistent locations on different maps (probably due to different coordinates). It would be good if the label could be placed at a consistent spot that's out of the way (e.g. top left corner of each map).

![Capture](https://user-images.githubusercontent.com/72491792/150047599-1fbb7d1b-9666-4a23-9346-9bb076e7f9a2.JPG)

### Save game and other files
[liberation_nextturn.zip](https://github.com/dcs-liberation/dcs_liberation/files/7897297/liberation_nextturn.zip)

**Comment by DanAlbert** [comment]
> No save file required for this one, I think.

FWIW, even when they're extremely obvious a save game is still helpful because it gives us a thing to verify. We can be very sure of what the fix is, but we don't want to check anything in without verifying the fix. We can generate our own game to verify fixes sometimes, but it's a much better vote of confidence if I know that I've fixed _your_ game. That's why we ask for save games. It's the difference between knowing if I've fixed _a_ bug or _the_ bug.
